pystiche_papers/johnson_alahi_li_2016/_utils.py

```diff
@@ -42,10 +42,29 @@ def postprocessor() -> transforms.CaffePostprocessing:
 
 
 def multi_layer_encoder(impl_params: bool = True,) -> enc.VGGMultiLayerEncoder:
+    r"""Multi-layer encoder from :cite:`JAL2016`.
+
+    Args:
+        impl_params: If ``True``, use the parameters used in the reference
+            implementation of the original authors rather than what is described in
+            the paper. For details see below.
+
+    If ``impl_params`` is ``True`` , an external instead of an internal preprocessing of
```

Comment:
We can't say anything here about the external preprocessing. We should simply state that no internal preprocessing is used.